app/routes/submit.js

```diff
@@ -0,0 +1,52 @@
+const { MessageSender } = require('ffc-messaging')
+const { GET, POST } = require('../constants/http-verbs')
+const { USER } = require('ffc-auth/scopes')
+
+module.exports = [
+  {
+    method: GET,
+    path: '/submit',
+    options: {
+      auth: { strategy: 'jwt', scope: [USER] }
+    },
+    handler: async (_request, h) => {
+      return h.view('submit')
+    }
+  },
+  {
+    method: POST,
+    path: '/submit',
+    options: {
+      auth: false
+    },
+    handler: async (request, h) => {
+      try {
+        const sender = new MessageSender({
+          useCredentialChain: false,
+          host: process.env.MESSAGE_HOST,
+          username: process.env.MESSAGE_USER,
+          password: process.env.MESSAGE_PASSWORD,
+          address: process.env.PROCESSOR_TOPIC_ADDRESS
```

Comment:
Given this is only a prototype not going to be too picky about this but mentioning it anyway.

Typically would keep the environment variables in a common config component that checks for their presence when the application starts up.  That way if any are missing the application crashes and therefore at deployment time, you know you've got an issue.  Rather than app starting fine and then there being a failure when the user submits a particular form.

Reply:
Yeah definitely want to refactor this, but planning on doing this on separate PR to better understand how that config directory is set up in the repos in general.